src/tests/ftest/util/mpio_utils.py

```diff
@@ -137,13 +138,23 @@ def run_llnl_mpi4py(self, basepath, hostfile, pool_uuid, test_repo,
                                                       "/test_io_daos.py"):
             test_cmd = "mpiexec -n {} --hostfile {} python test_io_daos.py"\
                            .format(client_processes, hostfile)
+        elif test_name == "hdf5" and (os.path.isfile(test_repo + "/testphdf5")
+                                      and os.path.isfile(test_repo
+                                                         + "/t_shapesame")):
+            test_cmd = ("echo ***Running testhdf5*** ;" +
+                        " mpirun -np {} --hostfile {} ./testphdf5 ;"\
+                        .format(client_processes, hostfile) +
+                        "echo ***Running t_shapesame*** ;" +
+                        "mpirun -np {} --hostfile {} ./t_shapesame"\
+                        .format(client_processes, hostfile))
         else:
             raise MpioFailed("Wrong test name or test repo location specified")
 
         run_cmd = cd_cmd + ";" + env_variables[0] + ";" + env_variables[1] \
                   + ";" + env_variables[2] + ";" + env_variables[3] + ";" \
                   + env_variables[4] + ";" + env_variables[5] + ";" \
-                  + env_variables[6] + ";" + env_variables[7] + ";" + test_cmd
+                  + env_variables[6] + ";" + env_variables[7] + ";" \
```

Comment:
Might I suggest a different way to handle the exporting of the environment variables:

env_variables = {
    "PATH={}/bin:$PATH".format(self.mpichinstall),
    "LD_LIBRARY_PATH": "{}/lib:$LD_LIBRARY_PATH".format(self.mpichinstall),	                  
    "CRT_ATTACH_INFO_PATH": "{}/install/tmp/".format(basepath),	                             
    "MPI_LIB: "",
    "DAOS_SINGLETON_CLI": 1,	
    "MPIO_USER_PATH": "daos:",	                      
    "DAOS_POOL": pool_uuid,	          
    "DAOS_SVCL": 0,	                  
    "HDF5_PARAPREFIX": "daos:",
}

run_cmd = ";".join([cd_cmd, ";".join(["export {}={}".format(key, val) for key, val in env_variables.items()]), test_cmd]

Reply:
Done. Thanks for the suggestion.